From a3a717668b8158c555490921722c04ea6c07bf1a Mon Sep 17 00:00:00 2001
From: Green Wan <green.wan@sifive.com>
Date: Mon, 28 Jun 2021 19:13:08 +0800
Subject: [PATCH 02/16] drivers: clk: sifive: fu740-prci: replace 'pciaux' with
 'pcieaux'

Replace 'pciaux' with 'pcieaux', including name string and function
prefix. The old name string, 'pciaux', might cause an error if PCIe
driver is changed to use clk_get_by_name() with 'pcieaux' to get
clock.

Signed-off-by: Green Wan <green.wan@sifive.com>
Reviewed-by: Leo Yu-Chi Liang <ycliang@andestech.com>
---
 drivers/clk/sifive/fu740-prci.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/clk/sifive/fu740-prci.c b/drivers/clk/sifive/fu740-prci.c
index 9a642c1..b025050 100644
--- a/drivers/clk/sifive/fu740-prci.c
+++ b/drivers/clk/sifive/fu740-prci.c
@@ -20,7 +20,7 @@
 #include "sifive-prci.h"
 #include <asm/io.h>
 
-int sifive_prci_fu740_pciauxclk_enable(struct __prci_clock *pc, bool enable)
+int sifive_prci_fu740_pcieauxclk_enable(struct __prci_clock *pc, bool enable)
 {
 	struct __prci_wrpll_data *pwd = pc->pwd;
 	struct __prci_data *pd = pc->pd;
@@ -98,7 +98,7 @@ static const struct __prci_clock_ops sifive_fu740_prci_hfpclkplldiv_clk_ops = {
 };
 
 static const struct __prci_clock_ops sifive_fu740_prci_pcieaux_clk_ops = {
-	.enable_clk = sifive_prci_fu740_pciauxclk_enable,
+	.enable_clk = sifive_prci_fu740_pcieauxclk_enable,
 };
 
 /* List of clock controls provided by the PRCI */
@@ -150,7 +150,7 @@ struct __prci_clock __prci_init_clocks_fu740[] = {
 		.ops = &sifive_fu740_prci_hfpclkplldiv_clk_ops,
 	},
 	[PRCI_CLK_PCIEAUX] {
-		.name = "pciaux",
+		.name = "pcieaux",
 		.parent_name = "",
 		.ops = &sifive_fu740_prci_pcieaux_clk_ops,
 		.pwd = &__prci_pcieaux_data,
-- 
2.7.4

